crds/jwst/specs/fgs_apcorr.rmap

```diff
@@ -0,0 +1,73 @@
+header = {
+    'classes' : ('Match', 'UseAfter'),
+    'derived_from' : 'Hand made 2019-10-22',
+    'file_ext' : '.fits',
+    'filekind' : 'APCORR',
+    'filetype' : 'APCORR',
+    'instrument' : 'FGS',
+    'mapping' : 'REFERENCE',
+    'name' : 'jwst_fgs_apcorr.rmap',
+    'observatory' : 'JWST',
+    'parkey' : (('META.INSTRUMENT.DETECTOR',), ('META.OBSERVATION.DATE', 'META.OBSERVATION.TIME')),
+    'sha1sum' : '88283a02fec824ae480cf0cc3bb41ca26577dcc3',
```

Comment:
I was wondering [how on earth](https://github.com/spacetelescope/crds/blob/6f0a16f2848709532cd6111aef585ff6905b3205/crds/core/rmap.py#L399) you got this to match...

Reply:
Do you mean the actual value being in the repo? Was just [following the directions](https://innerspace.stsci.edu/display/CRDS/Adding+New+Reference+Types). The `install` script runs the hash calculator, which I forget the command at the moment. I run the install locally, so things are all set.

Reply:
Yea, I just couldn't figure out how it was possible to checksum a file that also contains the checksum string.